tool/tsh/app.go

```diff
@@ -228,30 +232,40 @@ func onAppConfig(cf *CLIConf) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	conf, err := formatAppConfig(tc, profile, app.Name, app.PublicAddr, cf.Format, "")
+	conf, err := formatAppConfig(tc, profile, app.Name, app.PublicAddr, cf.Format, "", cf.InsecureSkipVerify)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 	fmt.Print(conf)
 	return nil
 }
 
-func formatAppConfig(tc *client.TeleportClient, profile *client.ProfileStatus, appName, appPublicAddr, format, cluster string) (string, error) {
+func formatAppConfig(tc *client.TeleportClient, profile *client.ProfileStatus, appName, appPublicAddr, format, cluster string, insecure bool) (string, error) {
 	var uri string
 	if port := tc.WebProxyPort(); port == teleport.StandardHTTPSPort {
 		uri = fmt.Sprintf("https://%v", appPublicAddr)
 	} else {
 		uri = fmt.Sprintf("https://%v:%v", appPublicAddr, port)
 	}
-	curlCmd := fmt.Sprintf(`curl \
-  --cacert %v \
+
+	var curlCmd string
+	if insecure {
+		curlCmd = fmt.Sprintf(`curl -k \
```

Comment:
```suggestion
		curlCmd = fmt.Sprintf(`curl --insecure \
```

Since the user is probably going to copy this anyway, the more verbose version should be preferred. This gives them more of a warning that what they're doing is not secure.

Reply:
Sounds good, thank you!